docs/styles.md

```diff
@@ -4,6 +4,7 @@
   - [Styling Content for RAISE](#styling-content-for-raise)
 - Tables
   - [Text Heavy Tables](#text-heavy-table)
+  - [Adjustable Column Text Heavy Tables](#text-heavy-adjusted-table)
```

Comment:
This anchor link doesn't seem to work. Maybe it needs to be `#adjustable-column-text-heavy-table` or we should update the title align with this anchor :man_shrugging: